flowapi/tests/unit/test_run_query.py

```diff
@@ -42,13 +42,13 @@
     "query, expected_msg",
     [
         (
-            {"query_kind": "daily_location", "params": {"date": "2016-01-01"}},
+            {
+                "query_kind": "spatial_aggregation",
+                "locations": {"query_kind": "daily_location", "date": "2016-01-01"},
+            },
             "Aggregation unit must be specified when running a query.",
         ),
-        (
-            {"params": {"date": "2016-01-01"}},
-            "Query kind must be specified when running a query.",
-        ),
+        ({"date": "2016-01-01"}, "Query kind must be specified when running a query."),
```

Comment:
Do we want/need an additional test case where the toplevel query kind is `"spatial_aggregation"` but then the query kind is missing inside the `"locations"`? When I try this locally I currently get a 401 but presumably we want 400 as for the other cases in this test?